packages/backend-plugin-api/api-report.md

```diff
@@ -24,6 +25,64 @@ import { Readable } from 'stream';
 import { Request as Request_2 } from 'express';
 import { Response as Response_2 } from 'express';
 
+// @public
+export type AuditEvent = [
```

Comment:
backend-plugin-api is kinda a big pile of things, so we've settled on prefixing service specific types according to the service name. You could make all of these types start with `EventAuditor*` - or `Auditor*` if we decide to shorten the service name